src/werkzeug/debug/shared/debugger.js

```diff
@@ -1,3 +1,25 @@
+function fadeOut(el){
+    el.style.opacity = 1;
+
+    (function fade() {
+        if ((el.style.opacity -= .1) < 0) el.style.display = "none";
+        else requestAnimationFrame(fade);
+    })();
+};
+
+function fadeIn(el, display){
```

Comment:
```suggestion
function fadeIn(el, display) {
```
Can we use a more descriptive name than `el`?